sdk/logical/request.go

```diff
@@ -364,21 +364,21 @@ type Operation string
 const (
 	// The operations below are called per path
 	CreateOperation         Operation = "create"
-	ReadOperation                     = "read"
-	UpdateOperation                   = "update"
-	PatchOperation                    = "patch"
-	DeleteOperation                   = "delete"
-	ListOperation                     = "list"
-	ScanOperation                     = "scan"
-	HelpOperation                     = "help"
-	AliasLookaheadOperation           = "alias-lookahead"
-	ResolveRoleOperation              = "resolve-role"
-	HeaderOperation                   = "header"
+	ReadOperation           Operation = "read"
+	UpdateOperation         Operation = "update"
```

Comment:
Strictly I think this is a breaking change but that's fine.

Reply:
Oof, you're right, I hadn't thought about that 😬